.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: generic
+services:
+  - docker
+
+notifications:
+  email:
+    on_success: never
+    on_failure: never
+
+env:
+  global:
+    - UPSTREAM_WORKSPACE=file
+    - ROSINSTALL_FILENAME=opp.rosinstall
+    - ROS_REPO=ros
+    - CMAKE_ARGS="-DCMAKE_BUILD_TYPE=Debug"
+    - ROSDEP_SKIP_KEYS="libqt5sql5-mysql"
+
+jobs:
+  include:
+    - os: linux
+      dist: bionic
+      language: cpp
+      env:
+      - ROS_DISTRO="melodic"
+      - CLANG_FORMAT_CHECK=file
+      - CLANG_FORMAT_VERSION=8
+      - DOCKER_IMAGE=lharmstrong/tesseract:melodic
```

Comment:
Why are we using a tesseract docker image here?

Reply:
It probably shouldn't. I can remove this.

Reply:
Maybe we can try the regular ROS Melodic image like you mentioned [here](https://github.com/swri-robotics/Toolpath-Offline-Planning/pull/10#issuecomment-575207519)